### ISSUE TITLE
Reverse numbering

### ISSUE THREAD

**Issue by austinj** [btskinner/tex_cv#3]
I'd like my numbered labeling scheme for publication sections to be descending from the most recent article, such that instead of being numbered [1], [2], [3] starting with the most recent publication, it's instead numbered [3], [2]. [1]. I've been banging my head against StackOverflow trying to implement a solution, but no luck yet.

**Comment by btskinner** [comment]
It's been a while since I've looked at the split branch that allows for numbered entries. I'm assuming that doesn't number the way you want?

**Comment by austinj** [comment]
No; unfortunately, it starts numbering at 1 with the most recent entry
instead of starting with 1 at the oldest entry.

**Comment by btskinner** [comment]
Sorry I've been busy. I'll try to figure out an option for this when I have time. My guess is it's a straightforward adjustment to the numbered entries branch to reverse order. If you have an idea and can do a PR, I'm glad to look. Otherwise, I'll get to it as soon as I can.

**Comment by austinj** [comment]
Thanks; I've tried quite a few things off Stack Overflow, but I just
haven't been able to make them work.
